src/main/java/com/flowable/flowableboot/repository/UmsTaskRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.flowable.flowableboot.repository;
+
+import com.flowable.flowableboot.model.UmsTask;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
+
+import java.util.List;
+
+public interface UmsTaskRepository extends CrudRepository<UmsTask, Integer> {
+    List<UmsTask> findByAssignee (@Param("assignee")String assignee);
```

Comment:
Is there a reason that @Param in necessary here?  I'm not familiar with when this is required - I would assume maybe when the column name in the database doesn't match the variable name in the java object.

Reply:
No, it is only required in the case that you mentioned with a difference between the variable and the request parameter.

Reply:
Ah, gotcha!  Yeah if they match it's not needed.  You can leave it there if you like.